lib/resources/submissions.js

```diff
@@ -297,14 +298,14 @@ module.exports = (service, endpoint) => {
               response.append('Content-Type', 'application/zip');
               return zipStreamFromParts(
                 // TODO: not 100% sure that these streams close right on crash.
-                streamBriefcaseCsvs(rows, fields, form.xmlFormId, selectValues, decryptor, false, options),
-                streamAttachments(attachments, decryptor),
-                streamClientAudits(clientAudits, form, decryptor)
+                streamBriefcaseCsvs(s3, rows, fields, form.xmlFormId, selectValues, decryptor, false, options),
+                streamAttachments(s3, attachments, decryptor),
+                streamClientAudits(s3, clientAudits, form, decryptor)
```

Comment:
Don't pass `s3` to these functions - `X.streamForExport()` functions should fetch from s3 transparently.